nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-nar-loading-utils/src/test/java/org/apache/nifi/nar/TestNarLoader.java

```diff
@@ -33,20 +33,18 @@
 import java.util.Arrays;
 import java.util.List;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 
+
+@EnabledOnOs({ OS.MAC, OS.LINUX })
+@EnabledIfSystemProperty(named = "os.arch", matches = "x86_64")
```

Comment:
On further review, it seems better to use `DisabledOnOs` with `OS.WINDOWS` and use `DisabledIfSystemProperty` matching `aarch64`.